extensions/target-specific/chromium/blink/wtf-helpers/wtf-helpers.js

```diff
@@ -21,7 +21,13 @@ Loader.OnLoad(function() {
     }));
 
     DbgObject.AddTypeDescription(Chromium.RendererProcessType("WTF::StringImpl"), "Text", true, UserEditableFunctions.Create((wtfStringImpl) => {
-        return !wtfStringImpl.isNull() ? wtfStringImpl.idx(1).as("char", /*disregardSize*/true).string(wtfStringImpl.f("length_")) : "";
+        if (!wtfStringImpl.isNull()) {
+            return wtfStringImpl.f("is_8bit_").val()
+            .then((is8bit) => wtfStringImpl.idx(1).as(is8bit ? "char" : "wchar_t", /*disregardSize*/true))

```

Comment:
wchar_t is 4 byte on Linux -- would char16_t work?

Reply:
Good call! I wonder if we will need to update getTypeDescriptionFunctionIncludingBaseTypes in description.js to handle char16_t at some point.